app/assets/javascripts/home.js

```diff
@@ -0,0 +1,26 @@
+var settingsObj = {
+
+  processUserInput: function(cellClickEvent){
+    var val = settingsObj.getRowSize(cellClickEvent);
+    var max = settingsObj.updateForm(val)
+    settingsObj.updatePlaceholder(val, max);
+  },
+
+  getRowSize: function(cellClickEvent){
+    return cellClickEvent.target.value;
+  },
+
+  updateForm: function(val){
+    var bomb_count = parseInt(val * val * 0.75).toString();
```

Comment:
Nitpick: `parseInt(val * val * 0.75).toString()` could have more descriptive names for `val` and `0.75`.  If `0.75` doesn't change it might also be nice to make it a constant.